pkg/db/v3/namespace.go

```diff
@@ -0,0 +1,112 @@
+package v3
```

Comment:
(not a request destined for this PR, just noting a conversation) We discussed a potential improvement for the future, where the Grype-specific and Builder-specific concerns would be moved to those respective codebases, and what would be provided by Grype-DB would be squarely the data types, values, and simple value-producing functions that are common to both production and consumption of the database. In doing so, this code would be oblivious to changes in Grype needs and Builder needs.

("Builder" == grype-db-builder)